src/main/java/pl/trainingapp/exceptions/EntityNotFoundException.java

```diff
@@ -0,0 +1,6 @@
+package pl.trainingapp.exceptions;
+
+public class EntityNotFoundException extends Exception {
```

Comment:
This should be RuntimeException so we won't need to add throws EntityNotFound... to the method signature everywhere
https://www.baeldung.com/java-checked-unchecked-exceptions

Reply:
ok, changed